cypress/e2e/data-import/settings/create-new-action-profile-with-long-name.cy.js

```diff
@@ -0,0 +1,41 @@
+import getRandomStringCode from '../../../support/utils/genereteTextCode';
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import { FOLIO_RECORD_TYPE } from '../../../support/constants';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import ActionProfiles from '../../../support/fragments/data_import/action_profiles/actionProfiles';
+import Users from '../../../support/fragments/users/users';
+import ActionProfileView from '../../../support/fragments/data_import/action_profiles/actionProfileView';
+
+describe('data-import: Settings: Create a new action profile with a long name', () => {
+  let user;
+  const actionProfile = {
+    typeValue: FOLIO_RECORD_TYPE.INSTANCE,
+    name: `C2343 autotest action profile ${getRandomStringCode(160)}`,
+  };
+
+  before('create user', () => {
+    cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+      user = userProperties;
+      cy.login(user.username, user.password, {
+        path: SettingsMenu.actionProfilePath,
+        waiter: ActionProfiles.waitLoading,
+      });
+    });
+  });
+
+  after('delete test data', () => {
+    Users.deleteViaApi(user.userId);
+    ActionProfiles.deleteActionProfile(actionProfile.name);
+  });
+
+  it(
+    'C2343 Create a new action profile with name longer than 160 symbols',
```

Comment:
Should have "(TaaS)" at the end of description:
C2343 Create a new action profile with name longer than 160 symbols (folijet) (TaaS)

Reply:
added